src/routers/SwitchRouter.js

```diff
@@ -56,7 +56,11 @@ export default (routeConfigs, config = {}) => {
 
   function resetChildRoute(routeName) {
     const params =
-      routeName === initialRouteName ? initialRouteParams : undefined;
+      routeName === initialRouteName
+        ? initialRouteParams
```

Comment:
shouldn't this be `{...initalRouteParams, ...routeConfigs[routeName].params }`

Reply:
yeah, that's better )

Reply:
@ericvicenti , should we assign initialRouteParams to every child route? or ,
let "initalRouteParams" - for initial route, and simply add defaultParams to child route ?